app/views/application/_captcha.html.erb

```diff
@@ -0,0 +1,6 @@
+<% field_class = field_class || '' %>
```

Comment:
😐 Don't do this, don't create variables in a view, _and_ you are using it only once :P

Reply:
field_class.presence :)

Reply:
_And_ if `field_class` is `nil`, erb will do nothing with it :P